src/components/UserWallet.vue

```diff
@@ -14,10 +20,10 @@ import {
 import { useModalStore } from '@/store/modal'
 import { useRpcStore } from '@/store/rpc'
 import { useUserStore } from '@/store/user'
-import { HIDE_ON_RAMP } from '@/utils/constants'
 import { getImage } from '@/utils/getImage'
 import { isSupportedByOnRampMoney } from '@/utils/onrampmoney.ramp'
 import { getRequestHandler } from '@/utils/requestHandlerSingleton'
+import shrinkWalletAddress from '@/utils/shrinkWalletAddress'
```

Comment:
I guess we already have some `truncate` method, we can use that over here instead of creating a new utility